analysis_scripts/qc_scripts/pipeline_qc.md

```diff
@@ -0,0 +1,74 @@
+
+#### plotting article level stats
+
+Now lets do a quick plot of article specific stats. So far in the pipeline, we only have the body of the text, so lets just plot that and make sure that it is consistent over time and there are no outliers.
+
+<img src="pipeline_qc_files/figure-markdown_github/unnamed-chunk-3-1.png" style="display: block; margin: auto;" /><img src="pipeline_qc_files/figure-markdown_github/unnamed-chunk-3-2.png" style="display: block; margin: auto;" />
+
+## QC pipeline step 2: Check processed scrapy output
+
+    ## ### 2010 has a consistent number of articles
+    ## ### 2015 has a consistent number of articles
+    ## ### 2016 has a consistent number of articles
+    ## ### 2017 has a consistent number of articles
+    ## ### 2018 has a consistent number of articles
+    ## ### 2019 has a consistent number of articles
+    ## ### 2020 has a consistent number of articles
+
+#### plotting article level stats
+
+<img src="pipeline_qc_files/figure-markdown_github/figures-side-1.png" width="50%" /><img src="pipeline_qc_files/figure-markdown_github/figures-side-2.png" width="50%" />
+
+## QC pipeline step 3: Check coreNLP output
+
+    ## [1] "2010"
+    ## ### 2010 has a consistent number of articles
+    ## [1] "2015"
+    ## ### 2015 has a consistent number of articles
+    ## [1] "2016"
+    ## ### 2016 has a consistent number of articles
+    ## [1] "2017"
+    ## ### 2017 has a consistent number of articles
+    ## [1] "2018"
+    ## ### 2018 has a consistent number of articles
+    ## [1] "2019"
+    ## ### 2019 has a consistent number of articles
+    ## [1] "2020"
+    ## ### 2020 has a consistent number of articles
+
+#### plotting article level stats
+
+<img src="pipeline_qc_files/figure-markdown_github/unnamed-chunk-7-1.png" width="50%" /><img src="pipeline_qc_files/figure-markdown_github/unnamed-chunk-7-2.png" width="50%" /> <img src="pipeline_qc_files/figure-markdown_github/unnamed-chunk-8-1.png" width="30%" /><img src="pipeline_qc_files/figure-markdown_github/unnamed-chunk-8-2.png" width="30%" /><img src="pipeline_qc_files/figure-markdown_github/unnamed-chunk-8-3.png" width="30%" /> <img src="pipeline_qc_files/figure-markdown_github/unnamed-chunk-9-1.png" width="30%" /><img src="pipeline_qc_files/figure-markdown_github/unnamed-chunk-9-2.png" width="30%" /><img src="pipeline_qc_files/figure-markdown_github/unnamed-chunk-9-3.png" width="30%" />
+
+## QC pipeline step 4: Check coreNLP processed output
+
+#### plotting quote stats
+
+<img src="pipeline_qc_files/figure-markdown_github/unnamed-chunk-11-1.png" width="50%" /><img src="pipeline_qc_files/figure-markdown_github/unnamed-chunk-11-2.png" width="50%" /><img src="pipeline_qc_files/figure-markdown_github/unnamed-chunk-11-3.png" width="50%" /><img src="pipeline_qc_files/figure-markdown_github/unnamed-chunk-11-4.png" width="50%" /><img src="pipeline_qc_files/figure-markdown_github/unnamed-chunk-11-5.png" width="50%" />
```

Comment:
For the fifth graph whats going on with the inflated turquoise rectangle after processing coreNLP output for unknown gender types? Is because coreNLP isn't doing a great job at predicting female names?

Reply:
Ah, this is tricky because the boxplot is hiding the density of the points. 

I now add in the points, jittered a bit, to show that actually we are doing much better overall (in commit ff84491). There is only a few articles that have remaining unknown genders.
![Screen Shot 2021-02-02 at 10 46 57](https://user-images.githubusercontent.com/1885501/106641095-0fab5400-6544-11eb-8185-fc7c66d00131.png)

Overall, we would like to see either the same number or greater female/male quotes per article, but a decrease in the unknown quotes. I have now replaced the old plots with new ones, to better show this.